scripts/ci/create-docker-image.sh

```diff
@@ -16,73 +19,90 @@ VERSION=${TAG#v}
 
 [ -n "$VERSION" ] && DOCKER_TAG=$VERSION || DOCKER_TAG=latest
 
+sha1=$( git rev-parse --short=12 HEAD )
+: ${SHA1:=$sha1}
+$WITH_SHA1 && [ "$DOCKER_TAG" != latest ] && : ${DOCKER_TAG:+-$SHA1}
```

Comment:
would read easier if combined with previous expression (changed to `if` conditions).

Reply:
modified to make reading easier